content/community/getting-involved/developing/system.html

```diff
@@ -151,8 +151,11 @@ <h4>
 
 <p>Haiku has great support for its own file system, but most others are only available read-only. It is way better for interoperability with other systems to be able to write to these disks from Haiku.</p>
 <p>The goal of this project is to complete the existing support for one of the following filesystems in Haiku, working from the existing code base:</p>
+
 <ol>
 <li>ReiserFS <a href="https://cgit.haiku-os.org/haiku/tree/src/add-ons/kernel/file_systems/reiserfs">existing sources</a>, <a href="http://web.archive.org/web/20070717210450/http://www.namesys.com/X0reiserfs.html">official specifications</a>, <a href="http://p-nand-q.com/download/rfstool/reiserfs_docs.html">extra documentation on the FS layout</a></li>
+<li>UFS2 (as used in *BSD): <a href="https://review.haiku-os.org/q/owner:suhelmehta%2540outlook.com">Current implementation</a></li>
+<li>XFS</a>: <a href="https://git.haiku-os.org/haiku/tree/src/add-ons/kernel/file_systems/xfs">Current implementation</a></li>
```

Comment:
This should be removed from here until the existing patches (for both XFS and UFS) are merged, and in the case of UFS, work on reading is needed still, as it doesn't work quite right yet.